### DISCUSSION TITLE
配置 MFSU 后，EventEmitter  load 失败

### DISCUSSION THREAD

**Discussion by duobancc** [umijs/umi#7138] [General]
* win10

![image](https://user-images.githubusercontent.com/37232082/128584680-23ef921f-beb9-4476-9808-780028fbb78e.png)
![image](https://user-images.githubusercontent.com/37232082/128584702-dd8cd494-78ec-4f28-906d-5a437b1370db.png)

**Comment by lisiyu55** [comment]
我也遇到了相同的问题，请问最后怎么解决的呢？

**Reply by duobancc** [reply]
import Event from 'events'
